xc/xenops_server_xen.ml

```diff
@@ -1903,6 +1903,7 @@ module VGPU = struct
 				let emulator_pid =
 					match vgpu.implementation with
 					| GVT_g _ -> Device.Qemu.pid ~xs frontend_domid
+					| MxGPU _
```

Comment:
This is not right: `Device.Vgpu.pid` refers to DEMU, which is only for Nvidia. For MxGPU, we could do the same as for GVT_g and refer to QEMU.